index.html

```diff
@@ -2595,6 +2595,137 @@ <h3>Data Schemas</h3>
+          <code>timestamp</code> property if included MUST be a string
+          value of an [[XMLSCHEMA11-2]] combined date-time string.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          a property named <code>mediaType</code> that indicates the 
+          expected media type for the indicated <code>resource</code>.
+          If a <code>mediaType</code> is included it must be a valid
+          media type as listed in the 
+          <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+            IANA Media Types
+          </a> registry.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          additional properties.
+        </p>
+        <p>
+          In any object in the <code>credential.credentialSubject</code>
+          that contains a [[URL]], a property named
+          <code>integrity</code> may be included with the integrity
+          information as specified above, with the [[URL]] in the
+          <code>id</code> property. 
+        </p>
+        <p>
+          Any objects for which selective disclosure is desired SHOULD NOT
+          be included as an object in the <code>resourceIntegrity</code>
+          array.
+        </p>
+        <p>
+          Implementers SHOULD consult appropriate sources, such as the 
+          <a href="https://www.iana.org/assignments/named-information/named-information.xhtml">IANA
+          Named Information Hash Algorithm Registry</a> to ensure that they
+          are chosing a current and reliable hash algorithm.  At the time of
+          this writing `sha-256` SHOULD be considered the minimum strength
+          hash algorithm for use by implemnters.
+        </p>
+        <p class="issue">
+          The working group is discussing if we will adopt subresource
+          integrity as defined in [[SRI]] is adopted into the [[JSON-LD]]
+          specification as noted in that specifications <a
+          href="https://www.w3.org/TR/json-ld11/#security">current
+          security considerations</a> of that specification, this hash in
+          the VC can serve as an additional check towards ensuring that a
+          cached context used when issuing the VC matches the remote
+          resource.
+        </p>
+        <p>
+          <aside
+            class="example"
+            title="resource integrity"
+          >
+            <p>An example of a resource integrity object referencing contexts</p>
+            <pre>
+              "resourceIntegrity": [{
+                "id": "https://www.w3.org/ns/credentials/v2",
+                "integrity": "sha384-lHKDHh0msc6pRx8PhDOMkNtSI8bOfsp4giNbUrw71nXXLf13nTqNJoRp3Nx+ArVK", 
+              },{  
+                "id": "https://www.w3.org/ns/credentials/examples/v2",  
+                "integrity": "sha384-zNNbQTWCSUSi0bbz7dbua+RcENv7C6FvlmYJ1Y+I727HsPOHdzwELMYO9Mz68M26", 
+              }]
```

Comment:
Bikeshedding ahead ...

Since `integrity` is going to be a globally scoped thing in our work here (not a consideration that the SRI spec had to deal with), I think this is actually where we should probably use the name `resourceIntegrity`. And for what `resourceIntegrity` is doing now, it's actually just indicating that there's some related thing without saying what the specific relationship is.

So what if we do this:

```suggestion
              "relatedResource": [{
                "id": "https://www.w3.org/ns/credentials/v2",
                "resourceIntegrity": "sha384-lHKDHh0msc6pRx8PhDOMkNtSI8bOfsp4giNbUrw71nXXLf13nTqNJoRp3Nx+ArVK", 
              },{  
                "id": "https://www.w3.org/ns/credentials/examples/v2",  
                "resourceIntegrity": "sha384-zNNbQTWCSUSi0bbz7dbua+RcENv7C6FvlmYJ1Y+I727HsPOHdzwELMYO9Mz68M26", 
              }]
```

I think that should make a collision with `resourceIntegrity` less likely when being used in something like the other example:

```
    "credentialSubject": {
      "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
      "image": {
          "id": "https://university.example/images/58473",
          "resourceIntegrity": "sha384-ZfAwuJmMgoX3s86L7x9XSPi3AEbiz6S/5SyGHJPCxWHs5NEth/c5S9QoS1zZft+J"
      },
      ...
    }
```

The resulting statements, respectively, will also make more sense I think:

```
A => relatedResource => B
B => resourceIntegrity => zNNbQTWCSUSi0bbz7dbua+RcENv7C6FvlmYJ1Y+I727HsPOHdzwELMYO9Mz68M26
```

And:

```
A => image => B
B => resourceIntegrity => sha384-ZfAwuJmMgoX3s86L7x9XSPi3AEbiz6S/5SyGHJPCxWHs5NEth/c5S9QoS1zZft+J
```

Reply:
@mprorock,

> question - any reason you are opposed to aligning with the definition and attribute name in SRI

Yes, the [comment at the top of my review](https://github.com/w3c/vc-data-model/pull/1140#discussion_r1231602134) was meant to give my reasoning for that:

> Since `integrity` is going to be a globally scoped thing in our work here (not a consideration that the SRI spec had to deal with), I think this is actually where we should probably use the name `resourceIntegrity`.

In other words, since we will be defining a globally scoped term for use anywhere in the graph as another property of any subject (which is the right approach, IMO, and works with selective disclosure), I think using just the name `integrity` is more likely to create conflicts with the various data people may model in their VCs. It would be best for us to avoid that. That's not a consideration the SRI spec had to deal with, so we're just being more specific by using `resourceIntegrity` instead.